cosmos-sdk-proto/src/lib.rs

```diff
@@ -5,13 +5,18 @@
 //! worth considering that modules seem to show up and go away with every RC.
 //! TODO actually implement features tag based compilation
 
-#![warn(trivial_casts, trivial_numeric_casts, unused_import_braces)]
+#![cfg_attr(docsrs, feature(doc_cfg))]
```

Comment:
The attribute is used to toggle-on `doc_cfg`, a nightly-only feature on https://docs.rs (which uses nightly by default)

See `doc(cfg(...))` attributes below and screenshots for how it's used.